### ISSUE TITLE
🐛 Biome fails to start

### ISSUE THREAD

**Issue by denivyruck** [biomejs/biome#10616]
### Environment information

Message received that biome in VSCode failed to start 5 times in the last 3 minutes (or something).

```bash
CLI:
  Version:                      1.9.3
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v20.18.2"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "pnpm/10.14.0"

Biome Configuration:
  Error:                        Found an unknown key `includes`.
  Error:                        Found an unknown key `assist`.
  Status:                       Loaded with errors
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

</details>

### What happened?

2026-06-12 12:54:21.411 [info] [Error - 12:54:21 PM] Server process exited with code 0.
2026-06-12 12:54:21.474 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:77:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

2026-06-12 12:54:21.486 [info] [Error - 12:54:21 PM] Server process exited with code 0.
2026-06-12 12:54:21.487 [info] [Info  - 12:54:21 PM] Connection to server got closed. Server will restart.
2026-06-12 12:54:21.487 [info] true
2026-06-12 12:54:21.562 [info] Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue:

Source Location: crates/biome_service/src/settings.rs:77:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.

### Expected result

Biome launches successfully regardless of workspace or not

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
Your Biome CLI looks too outdated (the rage output shows 1.9.3 and the latest is 2.5.0 as of today). Try the latest version and report with the updated info if the issue persists.

**Comment by ematipico** [comment]
There are clear errors 

```
Biome Configuration:
  Error:                        Found an unknown key `includes`.
  Error:                        Found an unknown key `assist`.
  Status:                       Loaded with errors
```

You have a modern configuration, but you're running an old CLI.
